src/Adyen/Config.php

```diff
@@ -150,6 +150,14 @@ public function getAdyenPaymentSource()
 		return isset($this->data['adyenPaymentSource']) ? $this->data['adyenPaymentSource'] : null;
 	}
 
+    /**
+     * @return array|null an array with 'name' and 'version'
+     */
+    public function getMerchantApplication()
+    {
+        return isset($this->data['merchantApplication']) ? $this->data['merchantApplication'] : null;
```

Comment:
let's slowly migrate off of the data field and use proper string value for this. protected $merchantApplication;

Reply:
ignore it as this will mean for every param we added it will be a major release for our merchants what is not ideal

Reply:
we can deprecate the interface so we are more flexible in the future.